dd-sdk-android/src/main/kotlin/com/datadog/android/rum/webview/RumContextProvider.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.webview
+
+import com.datadog.android.core.internal.utils.devLogger
+import com.datadog.android.core.internal.utils.sdkLogger
+import com.datadog.android.rum.GlobalRum
+import com.datadog.android.rum.internal.domain.RumContext
+import com.datadog.android.rum.internal.domain.scope.RumApplicationScope
+import com.datadog.android.rum.internal.domain.scope.RumSessionScope
+import com.datadog.android.rum.internal.monitor.DatadogRumMonitor
+
+internal class RumContextProvider {
+
+    fun getRumContext(): RumContext? {
+        val appScope = (GlobalRum.get() as? DatadogRumMonitor)?.rootScope as? RumApplicationScope
+        val rumContext = (appScope?.childScope as? RumSessionScope)?.getRumContext()
```

Comment:
We have a `GlobalRum.getRumContext()` method that you can call that will always have the latest up to date context. This would be even better since you don't need to know that
- the active monitor needs to be a `DatadogRumMonitor`
- the monitor's root scope is a `RumApplicationScope`
- the app's scope has a child scope
- the app's child scope is a RumSessionScope.

Any change to our monitor implementation would ruin this

Reply:
Yes...in the first place I wanted to use that method, problem is that it is not handling the session update. The only method that does that is the one in the `SessionScope`

Reply:
Hmm… Maybe we should have a mecanism to do this without coupling your work with the implementation details, i.e. add a paramater to "update session if needed" to the `GlobalRum.getRumContext()` ?
Tell you what, use the standard `GlobalRum.getRumContext()` for now, and add a ticket to see if we can have a cleaner way to ensure a new session is updated if needed.

Reply:
Hmmm...that would make sense...is just that this method is public, I think we will need something more internal for that but I will think about something in the next PR.

Reply:
My bad, that method is not public so I could actually tweak it to update the session if needed. In the same time I would rather add a new separated method than to actually tweak that one as it is not part of the same context.